packaging/centos8/cloud.spec

```diff
@@ -78,7 +78,7 @@ Requires: ipmitool
 Requires: %{name}-common = %{_ver}
 Requires: iptables-services
 Requires: qemu-img
-Requires: haveged
```

Comment:
@mlsorensen can you make similar change for consistency across all packaging distros (el7, suse and debian), see 
https://github.com/apache/cloudstack/pull/6522/commits/3ee62e336055818d519c9bfa6aded3ddd69a43ef
https://github.com/apache/cloudstack/pull/6522/commits/a240c3ba5d3637a0965d4b1a7a13b9e45200f881

Reply:
is that sane for all distros?

Reply:
Yes rng-tools is available on all distros